launcherctl/_util.py

```
@@ -1,13 +1,13 @@
 
 
-def LauncherCtlException(Exception):
+class LauncherCtlException(Exception):
     pass
 
 
-def launcherctl(*args) -> str | None:
+def launcherctl(*args: str) -> str:
     try:
-        return subprocess.check_output(["/opt/bin/launcherctl"] + list(args))
+        return subprocess.check_output(["/opt/bin/launcherctl", *args], text=True)
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)